src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,9 +1,71 @@
 <script setup>
+import { ref } from 'vue'
+
+const sentence = ref('')
+const maxN = ref(3)
+const combinations = ref([])
+
+const handle = (event) => {
+  event.preventDefault()
+
+  // Reset combinations
+  combinations.value = []
+
+  const words = sentence.value.split(/\s+/)
+
+  for (let n = 1; n <= maxN.value; n++) {
+    const uniqueCombinations = new Set()
+
+    for (let i = 0; i <= words.length - n; i++) {
+      const combination = words.slice(i, i + n).join(' ')
+      uniqueCombinations.add(combination)
+    }
+
+    if (uniqueCombinations.size != 0) {
+      combinations.value.push(Array.from(uniqueCombinations).join(", "))
```

Comment:
We have to pay attention to the formatting of the code. For example, in line 20 we used single quotes, but in line 25 we used double quotes. I suggest you use single quotes.